src/components/DpDatetimePicker/DpDatetimePicker.vue

```diff
@@ -54,6 +56,15 @@ export default {
   },
 
   props: {
+    dataCy: {
+      type: Object,
+      required: false,
+      default: () => ({}),
+      validator: (prop) => {
+        return Object.keys(prop).every(key => ['date', 'time'].includes(key))
+      }
+    },
+
```

Comment:
I suggest to just use one Key and then append ":date" and ":time" in the template.

Reply:
Sorry, I don't understand what you mean here?
We definitely want two keys here `date & time`
need it in template above:
```
<dp-datepicker :data-cy="dataCy.date">
<dp-time-picker :data-cy="dataCy.time">
```
like in `DpDateRangePicker.vue`

Reply:
my suggestion:
```
<dp-datepicker :data-cy="`${dataCy}:date`">
<dp-time-picker :data-cy="`${dataCy}:time`">
```

```
dataCy: {
      type: String,
      required: false,
      default: 'pick'
    }
```

Reply:
Thanks, that's works and short too!
Done in:
https://github.com/demos-europe/demosplan-ui/pull/773/commits/96a02c3312b0f1656b7cd3029afdcb6ed2a55a70
https://github.com/demos-europe/demosplan-core/pull/2815/commits/747a77821971719de9dc028a234bda277814b1a4

Reply:
could you refactor it for DpDateRangePicker.vue, as well?

Reply:
Yes
Done in:
https://github.com/demos-europe/demosplan-ui/pull/773/commits/60a810aeb1e2c53f087d21df95ba7a0713bf47f5
https://github.com/demos-europe/demosplan-core/pull/2815/commits/38b6b806269d24b3a4cd5669a2b21ed68988dd5e